app/src/main/java/com/ncnf/feed/ui/FeedFragment.java

```diff
@@ -22,24 +21,26 @@
 import com.ncnf.event.EventActivity;
 import com.ncnf.event.EventDB;
 
-import java.util.ArrayList;
 import java.util.List;
 
-import static com.ncnf.Utils.DEBUG_TAG;
 import static com.ncnf.Utils.UUID_KEY;
 
 
 public class FeedFragment extends Fragment {
 
     private RecyclerView.LayoutManager lManager;
     private EventAdapter adapter;
-    private List<Event> eventList = new ArrayList<>();
-    private EventDB eventDB;
+    private List<Event> eventList;
     private static final String CHANNEL_NAME = "events_to_be_shown";
 
     public FeedFragment(EventDB eventDB){
         super();
-        this.eventDB = eventDB;
+        this.eventList = eventDB.toList();
+    }
+
+    public FeedFragment(List<Event> eventList){
```

Comment:
Check on the list?

Reply:
I'm guessing you mean a check that the list is not a null object? If that's what you're mean you're right and I'll change it right away